crates/uv-interpreter/src/discovery.rs

```diff
@@ -651,7 +651,7 @@ pub fn find_default_interpreter(cache: &Cache) -> Result<InterpreterResult, Erro
 /// the first available version.
 ///
 /// See [`find_interpreter`] for more details on interpreter discovery.
-#[instrument(skip_all, fields(?request))]
+#[instrument(skip_all, fields(request))]
```

Comment:
Honestly a little confused by this but I think `?`was making it a debug representation which is way more verbose than we need?